### PR TITLE
Some suggestions and a typo fix

### PR DESCRIPTION
**Pull request by kofa73**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -81,19 +81,23 @@ you are strongly advised to take a backup first.
   along with the traditional color-picker for manual sampling of neutral colors.
 
   Color calibration internally implements gamut-mapping and gamut-clipping,
-  which attempts to preserve the hue and luminance of pixels while avoiding imaginary and
-  out-of-gamut colors at the beginning of the pipeline, to improve the robustness of
+  which attempt to preserve the hue and luminance of pixels while avoiding imaginary and
+  out-of-gamut colors at the beginning of the pipeline, in order to improve the robustness of
```
Comment (kofa73) [inline RELEASE_NOTES.md:85]: If 'which' refers back to 'gamut-mapping' and 'gamut-clipping' (so there is more than one subject), the verb should be in plural form.

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -81,19 +81,23 @@ you are strongly advised to take a backup first.
...
   color-grading modules inside the pipeline. This is made necessary because
   white balancing will only push input out-of-gamut colors farther away,
-  and will noticeably help dealing with artifical colored lights (LED, stage lighting, etc.).
+  and gamut-mapping and -clipping will noticeably help dealing with artifical colored
+  lights (LED, stage lighting, etc.).
```
Comment (kofa73) [inline RELEASE_NOTES.md:89]: I think without repeating the subjects, the 'will noticeably help' part could be interpreted as having 'white balance' as the subject.

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -81,19 +81,23 @@ you are strongly advised to take a backup first.
...
   allows users to choose to keep using the white balance module
   to perform the chromatic adaptation for new edits ("legacy" worflow, the default)
   or to use the new color calibration instead ("modern" workflow).
   The modern workflow still uses the usual white balance module, although
   with different default settings, because input color profiles and
-  demosaicing need at least a rough white balancing early in the pipe.
+  demosaicing need at least a rough white balancing early in the pipe. The new color
+  calibration necessitates the use of the standard color matrix as the
+  input color profile, and the use of the 'camera referred' setting in the white
+  balance module.
```
Comment (kofa73) [inline RELEASE_NOTES.md:100]: This is my understanding, may be wrong.

File: RELEASE_NOTES.md (hunk 2 above)
Comment (TurboGit) [reply RELEASE_NOTES.md:89]: you are repeating **and** here, not really good to me but I'm not native English.

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -81,19 +81,23 @@ you are strongly advised to take a backup first.
   along with the traditional color-picker for manual sampling of neutral colors.
 
   Color calibration internally implements gamut-mapping and gamut-clipping,
-  which attempts to preserve the hue and luminance of pixels while avoiding imaginary and
```
Comment (paperdigits) [inline RELEASE_NOTES.md:84]: `attempts` is correct here.

File: RELEASE_NOTES.md (hunk 2 above)
Comment (paperdigits) [reply RELEASE_NOTES.md:89]: We don't really do the `-clipping` style of things in English. 

You should say `gamut-mapping and gamut-clipping` or `gamut-mapping/clipping`.

File: RELEASE_NOTES.md (hunk 3 above)
Comment (paperdigits) [reply RELEASE_NOTES.md:100]: should be `demoasicing needs`

**Comment by TurboGit** [comment]
@kofa73 : I'll close because RELEASE_NOTES.md is emptied now in master as the branch is created. If there is a PR it must be done from darktable-3.4.x branch or just report it and I'll fix. Thanks.

File: RELEASE_NOTES.md (hunk 2 above)
Comment (elstoc) [reply RELEASE_NOTES.md:89]: could even live without the hyphen

File: RELEASE_NOTES.md (hunk 3 above)
Comment (kofa73) [reply RELEASE_NOTES.md:100]: I was referring to Aurélien's text from https://discuss.pixls.us/t/introducing-color-calibration-module-formerly-known-as-channel-mixer-rgb/21227:
>Then, the default parameters of the color calibration will:
> - read the raw EXIF to fetch the scene white balance set by the camera,
> - adjust this setting with the reference white balance used in white balance module,
> - adjust this setting again with the input color profile in use (**only the standard matrix**).

I may have misunderstood.

**Pull request closed without merging**